docs/admin/runai-setup/authentication/sso.md

```diff
@@ -41,34 +41,45 @@ You can configure your IdP to map several IdP attributes:
 
 ## Step 1: UI Configuration
 
-1. Open the Administration User interface.
-2. Go to `Settings | General`.
-3. Turn on `Login with SSO`.
-4. Enter the administrator email.
-5. Select the SSO protocol. Choose `Saml 2` or `Open ID Connect`.
+1. Press the `Tools & Settings` then press `General`.
+2. Open the `Security` pane and press `Identity provider`.
+3. Select the SSO protocol. Choose `SAML 2` or `Open ID Connect`.
 
 !!! Note
     Use your SAML response file to fill in the fields below.
 
-For `Saml 2`:
+=== "SAML 2"
 
-   1. In the `Metadata XML Url` field, enter the URL to the XML Metadata file.
-   2. Find your identity provider's attribute names for `GID`, `GROUPS`, `SUPPLEMENTARYGROUPS` and `UID`. If they are not in line with the Run:ai defaults described in the table above, you can change them here.   
-   3. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.
-   4.  Press `Save`.
+    1. Choose `From computer` or `From URL`.
+    
+    	1. For `From computer`, press the `Metadata XML file` field, then select your file for upload. 
+    	2. For `From URL`, in the `Metadata XML Url` field, enter the URL to the XML Metadata file.
+    
+    2. Copy the `Redirect URL` and `Entity ID` for use with your identity provider if required.
+    3. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
+    4. When complete, press `Save`.
+    5. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.
```

Comment:
@jasonnovichRunAI I think this should go separatly because it is not under the identity provider / SAML